app/models/session.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class Session extends AppModel
+{
+    public static function setSession($session_name, $value)
+    {
+        $_SESSION["$session_name"] = $value;
+    }
+    public static function getSession($session_name)
+    {
+        return $_SESSION["$session_name"];
+    }
+}
```

Comment:
ah I see, this is because you extend to AppModel po which has a `set()` method :smile:
as this is session, it can be independent from AppModel po :bow: